ha/setup/ha_setup.py

```diff
@@ -344,6 +344,17 @@ def process(self):
             node_id = Conf.get(self._index, f"server_node.{machine_id}.node_id")
             self._confstore.set(f"{const.PVTFQDN_TO_NODEID_KEY}/{node_name}", node_id)
 
+        node_id = Conf.get(self._index, f"server_node.{machine_id}.node_id")
```

Comment:
create a separate private function for it and call it here.

Reply:
Agree

Reply:
"_update_node_map()" function created.